src/components/ModularTable/ModularTable.test.tsx

```diff
@@ -61,4 +61,16 @@ describe("ModularTable", () => {
       "Nothing here"
     );
   });
+
+  it("renders a row with footer content", () => {
+    const wrapper = shallow(
+      <ModularTable columns={columns} data={data} footer="This is a footer" />
+    );
+
+    const rowItems = wrapper.find("tbody TableRow");
+    expect(rowItems.length).toEqual(4);
+    expect(rowItems.last().find("TableCell").first().children().text()).toEqual(
```

Comment:
This is a bit finicky...you should be able to target the footer directly.

Reply:
As discussed in other comment, we can't really use `tfoot` for that so hard to target this last row other way - unless we introduce test-specific identifiers.